catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -119,7 +123,20 @@ public void init() {
         "/geofeature/suggestions",
         (req, res) -> {
           String query = req.queryParams("q");
-          List<Suggestion> results = this.featureService.getSuggestedFeatureNames(query, 10);
+          List<Suggestion> resultSuggestions =
+              this.featureService.getSuggestedFeatureNames(query, 10);
+          Map<String, Object> latLonLiteral = getLatLonLiteral(query);
+          if (latLonLiteral == null) {
+            return mapper.toJson(resultSuggestions);
+          }
+          Function<Suggestion, Map<String, Object>> suggestionToMap =
+              (s) -> ImmutableMap.of("id", s.getId(), "name", s.getName(), "geo", "");
```

Comment:
I would pull this into a function that way it's self documenting.

```java
  /**
   * Takes a suggestion and transforms it into a map.
   *
   * <p>Geo is set to empty string because...
   */
  private static Map<String, Object> suggestionToMap(Suggestion s) {
    return new ImmutableMap.Builder<String, Object>()
        .put("id", s.getId())
        .put("name", s.getName())
        .put("geo", "")
        .build();
  }
```

Then you can use it similar to how you already are:
```java
         final List<Map<String, Object>> resultMapList =
              resultSuggestions
                  .stream()
                  .map(QueryApplication::suggestionToMap)
                  .collect(Collectors.toList());